app/scripts/lib/country-telephone-info.js

```diff
@@ -70,7 +70,7 @@
          }
          return `+1${num}`;
        },
-       pattern: /^(?:\+?1)?\d{10,10}$/, // allow for a +1 or 1 prefix before the area code
+       pattern: /^(\+?1)?[2-9]\d{9,9}$/, // allow for a +1 or 1 prefix before the area code, area codes are all 2-9
```

Comment:
Fwiw, not sure how bothered about this we are but I ended up with this regex in the auth server, after digesting all of the rules in [1]:

```
/^1?([2-9][0-8][0-9])([2-9][0-9]{2})([0-9]{4})$/
```

[1] https://www.nationalnanpa.com/number_resource_info/index.html